charts/greptimedb-cluster/Chart.yaml

```diff
@@ -23,3 +23,8 @@ dependencies:
     version: "8.5.8"
     repository: https://grafana.github.io/helm-charts
     condition: grafana.enabled
+
+  - name: jaeger-all-in-one
+    version: 0.1.12
+    repository: https://raw.githubusercontent.com/hansehe/jaeger-all-in-one/master/helm/charts
```

Comment:
Should use the official jaeger helm-charts?
https://github.com/jaegertracing/helm-charts/tree/main/charts/jaeger

Reply:
I don't choose the officail version because it's too complicated. We don't use it for production usgae, only for trouble shooting purpose.